client/src/translations/es.json

```
@@ -127,8 +127,8 @@
   "filter_visualisations_by_dataset": "Filtrar visualizaciones por conjunto de datos",
   "flow_application_url": "URL de la aplicación Flow",
   "first_name": "Nombre",
-  "following_visualisation_will_be_deleted":"Las siguiente visualización será borrada",
-  "following_visualisations_will_be_deleted":"Las siguientes visualizaciones serán borradas",
+  "following_visualisation_will_be_deleted": "Las siguiente visualización será borrada",
```

```suggestion
  "following_visualisation_will_be_deleted": "La siguiente visualización será borrada",
```
I have seen a little mistake here: there should be no plural